Clarify CAP error message, add window border

ahmedalsudani opened the pull request:
@fjvallarino @tejohnso 

### Changes

- Move error messages out to Markdown files
- Clarify CAP error
- Add border around installer window

tejohnso commented on installer/ui/messages/cap-installed.md:
```diff
@@ -0,0 +1,9 @@
+We have detected a previous installation of Rise Chrome App Player. You need to uninstall it before it proceeding with this installation.
```
typo here "uninstall it before it"

ahmedalsudani replied:
Oops. Thanks

tejohnso commented:
Looks good @ahmedalsudani 
Should add a test that confirms messages are being loaded into the global.  

And @ByronDarlison you should check out the updated message for CAP [here](https://github.com/Rise-Vision/rise-launcher-electron/blob/fix/borders-messages/installer/ui/messages/cap-installed.md)

ByronDarlison commented:
@tejohnso @ahmedalsudani message looks great. Thanks!

fjvallarino commented:
:+1:

ahmedalsudani commented:
@tejohnso addressed the issues you raised. Let me know if the test should be moved up a directory (from `test/ui` to `test`) or if the `max-height` change would be problematic

tejohnso commented:
@ahmedalsudani Looks good to me :+1:
